test/functional/shared/cli/cli_test.go

```diff
@@ -66,31 +66,37 @@ func verifyRecipeCLI(ctx context.Context, t *testing.T, test shared.RPTest) {
 	// for AWS test and has the AWS scope which the environment created in this does not.
 	envName := test.Steps[0].RPResources.Resources[0].Name
 	recipeName := "recipeName"
-	recipeTemplate := "ghcr.io/testpublicrecipe/bicep/modules/testTemplate:v1"
-	templateKind := "bicep"
-	resourceType := "Applications.Datastores/mongoDatabases"
-	file := "testdata/corerp-redis-recipe.bicep"
+	recipe1 := "recipe1"
+	recipe2 := "recipe2"
+	registry := strings.TrimPrefix(functional.GetBicepRecipeRegistry(), "registry=")
+	version := strings.TrimPrefix(functional.GetBicepRecipeVersion(), "version=")
+	recipeTemplate := fmt.Sprintf("%s/recipes/local-dev/rediscaches:%s", registry, version)
+	envRecipeTemplateBicep := fmt.Sprintf("%s/test/functional/shared/recipes/corerp-redis-recipe:%s", registry, version)
+	envRecipeTemplateTerraform := "Azure/cosmosdb/azurerm"
+	templateKindBicep := "bicep"
+	templateKindTerraform := "terraform"
```

Comment:
Some suggestions for readability, let me know if they don't make sense.

```suggestion
        registry := strings.TrimPrefix(functional.GetBicepRecipeRegistry(), "registry=")
	version := strings.TrimPrefix(functional.GetBicepRecipeVersion(), "version=")
	
        recipeName := "recipeName"
        recipeTemplate := fmt.Sprintf("%s/recipes/local-dev/rediscaches:%s", registry, version)
        
	bicepRecipe := "recipe1"
	bicepRecipeTemplate := fmt.Sprintf("%s/test/functional/shared/recipes/corerp-redis-recipe:%s", registry, version)
	templateKindBicep := "bicep"

	terraformRecipe := "recipe2"
	terraformRecipeTemplate := "Azure/cosmosdb/azurerm"
	templateKindTerraform := "terraform"
```

Reply:
Updated to this